hack/builder/Dockerfile

```diff
@@ -1,19 +1,19 @@
 ARG ARCH
-FROM ${ARCH}/fedora:30
+FROM ${ARCH}/fedora:31
 
 COPY qemu-ppc64le-static /usr/bin/qemu-ppc64le-static
 
-ENV LIBVIRT_VERSION 5.1.0
+ENV LIBVIRT_VERSION 5.0.0
 
 ENV BAZEL_VERSION 1.2.1
 
 # Install packages
 RUN dnf install -y dnf-plugins-core && \
-    dnf copr enable -y @virtmaint-sig/virt-preview && \
     dnf copr enable -y vbatts/bazel && \
+    dnf copr enable -y @virtmaint-sig/for-kubevirt && \
     dnf -y install \
         libvirt-devel-${LIBVIRT_VERSION} \
-        bazel-${BAZEL_VERSION} \
+        bazel \
```

Comment:
The bazel version is also gone? Which bazel ersion do we get now? Please hardcode it to avoid surpises (which I had in the past).